tests/unittests/api/helpers/test_files.py

```diff
@@ -95,6 +102,46 @@ def upload_multi():
                 self.assertEqual(file_path, actual_file_path)
                 self.assertTrue(os.path.exists(file_path))
 
+    def test_create_save_resized_image(self):
+        with app.test_request_context():
+            image_url_test = 'https://cdn.pixabay.com/photo/2014/09/08/17/08/hot-air-balloons-439331_960_720.jpg'
+            width = 500
+            height = 200
+            aspect_ratio = False
+            upload_path = 'test'
+            resized_image_url = create_save_resized_image(image_url_test, width, aspect_ratio, height, upload_path, ext='png')
+            resized_image_file = app.config.get('BASE_DIR') + resized_image_url.split('/localhost')[1]
+            resized_width, resized_height = self.getsizes(resized_image_file)
+            self.assertTrue(os.path.exists(resized_image_file))
+            self.assertEqual(resized_width, width)
+            self.assertEqual(resized_height, height)
+
+    def test_create_save_image_sizes(self):
+        with app.test_request_context():
+            image_url_test = 'https://cdn.pixabay.com/photo/2014/09/08/17/08/hot-air-balloons-439331_960_720.jpg'
+            image_sizes_type = "event"
+            width_large = 1300
+            width_thumbnail = 500
+            width_icon = 75
+            image_sizes = create_save_image_sizes(image_url_test, image_sizes_type)
+
+            resized_image_url_large = image_sizes['large_image_url']
```

Comment:
I think we can also check here if the copy of the Orignal Image is being created or not.